colors/solarized.kak

```diff
@@ -1,24 +1,70 @@
 # solarized theme
 
+# Solarized colorscheme will start in light or dark background,
+# depending on the following option.
+# 
+decl -hidden str solarized_initially light
+
+# Map the following toggle function to an alias or a key sequence
+# for easy toggling between light and dark, eg,
+# map global user t :solarized-toggle-background<ret>
+# 
+def -docstring 'toggle solarized background between light and dark' -allow-override \
+solarized-toggle-background %{
+    %sh{
+        set_bg() printf %s\\n "set global solarized_background $1"
```

Comment:
When defining one-liner functions, I usually leave out the braces; I put them for multi-line functions. I'm using dash.